demo/lib/TransformationControls/addFooterControl.js

```diff
@@ -100,9 +100,39 @@ export default [
   null,
   (iframeWindow, iframeDocument, selectedValue, iframe) => {
     event.target.disabled = true
-    addFooterContainer(iframeWindow, iframeDocument)
-    addFooterMenu(iframeWindow, iframeDocument)
-    addFooterReadMore(iframeWindow, iframeDocument, iframe)
-    addFooterLegal(iframeWindow, iframeDocument)
+    if (iframeWindow.pagelib.c1) {
+      iframeWindow.pagelib.c1.Footer.add(
+        document,
+        'Knight Lore',  // TODO: expose the article for each closure
```

Comment:
@berndsi that's the only thing I wasn't able to do in this patch. Do you have any ideas on how to handle it?

Reply:
Which part? adding the footer or getting the title?

Reply:
Getting the title. The footer is added, although its UI is different from MCS.